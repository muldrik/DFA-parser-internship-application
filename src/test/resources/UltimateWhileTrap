c = 0
a = 1
d = 1
e = 1
r = 1
w = 1
while (c < 10)
    j = b
    w = 2
    m = w
    y = r
    a = 5
    z = e
    p = 0
    while (p < 5)
        b = a
        b = d
        a = 10
        d = 10
        u = 0
        w = 3
        q = 2
        x = q
        while u < 5
            r = r + 1
            e = e + 2
            q = 9
        end
        p = p + 1
    end
    c = c + 1
end