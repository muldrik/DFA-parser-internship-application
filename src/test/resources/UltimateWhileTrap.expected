a = 1
w = 1
j = b
m = w
y = r
z = e
b = a
w = 3
x = q
q = 9
